lib/recent.go

```diff
@@ -0,0 +1,89 @@
+package lib
```

Comment:
I actually started going down this route, before deciding against it to make the diffing easier (at least initially) and only _just_ removed the draft file :D 
![Screenshot from 2024-05-27 08-38-04](https://github.com/warrensbox/terraform-switcher/assets/1266262/22c740f2-5a53-43b8-bafe-74d45c8cbc91)